src/mail/view/MailView.ts

```diff
@@ -376,6 +376,17 @@ export class MailView extends BaseTopLevelView implements TopLevelView<MailViewA
 		return this.viewSlider
 	}
 
+	handleBackButton(): boolean {
+		if (this.viewSlider.focusedColumn === this.mailColumn) {
```

Comment:
do we need to manually handle focusing other columns? I think I would just check that if we are in multiselect that we do unselect, otherwise returning `false` would do the default behavior already?

Reply:
Good spotting, just handling the selection works fine. Commited now.